### PR TITLE
box-cleanup

### PR DESCRIPTION
**Pull request by emilyJLin95**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,6 +37,7 @@ build/Release
 
 # Dependency directories
 node_modules/
+packages/truffle/node_modules
```
Comment (Montoya) [inline .gitignore:40]: The above node_modules/ doesn't work for all directories? 

Can we use wildcards to make sure it will work across all directories?

**Comment by Montoya** [comment]
I think you need to run `yarn lint:fix`

File: .gitignore (hunk 1 above)
Comment (emilyJLin95) [reply .gitignore:40]: oh I'm big dumb

sorry usually my VS Code is configured to grey out files that are .gitignored, but for some reason it wasn't

Comment (emilyJLin95) [reply .gitignore]: addressed!

**Comment by Montoya** [comment]
LGTM

**Review by Montoya — APPROVED** [review]

**Pull request merged**